src/dlgTriggerEditor.cpp

```diff
@@ -3128,8 +3161,8 @@ void dlgTriggerEditor::saveTrigger()
         else if( _type == 5 ) regexPropertyList << REGEX_LINE_SPACER;
         else if( _type == 6 ) regexPropertyList << REGEX_COLOR_PATTERN;
     }
-    QString script = mpSourceEditor->toPlainText();
 
+    QString script = mpSourceEditorEdbeeDocument->text();
```

Comment:
Actually, the old editor was a QPlainText widget, not QTextEdit. There are traces of a QTextEdit though, maybe it used to be one.